scripts/manage/landing_page_base.md

```diff
@@ -114,6 +114,9 @@ accounts) must be pushed and applied to your deployment to take effect:
 
 ### Add Spinnaker account for GKE
 
+This script grants the Spinnaker deployment's Google Service Account the required
```

Comment:
I think we use "Spinnaker instance" elsewhere?

Reply:
Seems like it might be better to use "instance" in all instances (unless there's a subtle difference in meaning that I'm not getting) to avoid confusion between the deployed spinnaker and deployed applications.